content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -120,6 +120,7 @@ different Kubernetes components.
 | `HPAScaleToZero` | `false` | Alpha | 1.16 | |
 | `IdentifyPodOS` | `false` | Alpha | 1.23 | 1.23 |
 | `IdentifyPodOS` | `true` | Beta | 1.24 | |
+| `IdentifyPodOS` | `true` | GA | 1.25 | |
```

Comment:
Change these as the following and move it to the second table for graduated/deprecated features.
```suggestion
| `IdentifyPodOS` | `true` | Beta | 1.24 | 1.24 |
| `IdentifyPodOS` | `true` | GA | 1.25 | |
```